File: hw/basurto/u3/hw26Observer/hw26Observer.js/view/stockView.js
export class StockView {
    static printNotification(investorName, stock, args) {
        console.log(`Notified observer ${investorName}`);
        if (typeof args === 'string') {
            console.log(`The symbol of ${stock.symbol} changed to: ${args}`);
        } else if (typeof args === 'number') {
            console.log(`The price of ${stock.symbol} changed to: ${args.toFixed(2)}`);
        }
    }
}

File: hw/basurto/u3/hw26Observer/hw26Observer.java/Observer/src/main/java/ec/edu/espe/Observer/view/StockView.java

package ec.edu.espe.Observer.view;
import ec.edu.espe.Observer.model.Stock;
/**
 *
 * @author Esteban Basurto, CodeBreakers, @ESPE
 */
public class StockView {
    public static void printNotification(String investorName, Stock stock, Object args) {
        System.out.println("Notified observer " + investorName);
        if (args instanceof String) {
            System.out.println("The symbol of " + stock.getSymbol() + " changed to: " + args);
        } else if (args instanceof Double) {
            System.out.println("The price of " + stock.getSymbol() + " changed to: " + args);
        }
    }
}
